src/ert/scheduler/lsf_driver.py

```diff
@@ -463,9 +485,11 @@ async def _log_bhist_job_summary(self, job_id: str) -> None:
         )
         logger.info(f"Output from bhist -l: {process_message}")
 
-    async def _poll_once_by_bhist(self, missing_job_ids: Iterable[str]) -> _Stat:
+    async def _poll_once_by_bhist(
+        self, missing_job_ids: Iterable[str]
+    ) -> Dict[str, AnyJob]:
         if time.time() - self._bhist_cache_timestamp < self._bhist_required_cache_age:
-            return _Stat(**{"jobs": {}})
+            return _parse_jobs_dict({})
```

Comment:
Hmm, not sure why we did not do  `return {}` before to avoid the dummy function call?

Reply:
Should we just return empty dict rather than the function call now?

Reply:
It is arguably equivalent, so I think yes.